public/js/chooseAll-answer.js

```diff
@@ -0,0 +1,10 @@
+var chooseAllMCAnswerCount = 4; // default number of options
+
+/**
+* Adds an option to the multiple choice
```

Comment:
change `multiple choice` to `choose all options`

Reply:
or `choose all that apply`

Reply:
done